apis/compute/v1beta1/referencers.go

```diff
@@ -71,6 +71,27 @@ func (mg *GlobalAddress) ResolveReferences(ctx context.Context, c client.Reader)
 	return nil
 }
 
+// ResolveReferences of this Address
+func (mg *Address) ResolveReferences(ctx context.Context, c client.Reader) error {
+	r := reference.NewAPIResolver(c, mg)
+
+	// Resolve spec.forProvider.network
+	rsp, err := r.Resolve(ctx, reference.ResolutionRequest{
+		CurrentValue: reference.FromPtrValue(mg.Spec.ForProvider.Network),
+		Reference:    mg.Spec.ForProvider.NetworkRef,
+		Selector:     mg.Spec.ForProvider.NetworkSelector,
+		To:           reference.To{Managed: &Network{}, List: &NetworkList{}},
+		Extract:      NetworkURL(),
+	})
+	if err != nil {
+		return errors.Wrap(err, "spec.forProvider.network")
+	}
+	mg.Spec.ForProvider.Network = reference.ToPtrValue(rsp.ResolvedValue)
+	mg.Spec.ForProvider.NetworkRef = rsp.ResolvedReference
```

Comment:
The only missing piece to close this PR is add the resolvers to Subnetwork as well. You can check [this example](https://github.com/crossplane/provider-gcp/blob/469c003ddee13267be0552a2fcdf400ca725e489/apis/container/v1beta2/referencers.go#L45-L76) if you need guidance.

Reply:
I added the resolvers for subnetworks for GlobalAddress and Address now. I hope that was right.